contracts/modules/AaveWithdrawModule.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "interfaces/gnosis/IGnosisSafe.sol";
+import {DataTypes, IAaveV3Pool} from "interfaces/aave/IAaveV3Pool.sol";
+import "interfaces/aave/IAToken.sol";
+
+import {BaseModule} from "../BaseModule.sol";
+
+contract AaveWithdrawModule is BaseModule {
+    ////////////////////////////////////////////////////////////////////////////
+    // CONSTANTS
+    ////////////////////////////////////////////////////////////////////////////
+
+    IAaveV3Pool AAVE_POOL = IAaveV3Pool(0x7b5C526B7F8dfdff278b4a3e045083FBA4028790);
+
+    ////////////////////////////////////////////////////////////////////////////
+    // EVENTS
+    ////////////////////////////////////////////////////////////////////////////
+
+    event EmergencyWithdraw(address asset, uint256 amount, address signer, uint256 timestamp);
+
+    function withdraw(IGnosisSafe safe, address collateral) external isSigner(safe) {
```

Comment:
would suggest to avoid the standard `withdraw` here, maybe `aaveV3Withdraw`? this would also open up adding v1 and v2 funcs into this module later

Reply:
ah and again remove `safe` param to reduce complexity of call for signer.

Reply:
also, can we add a docstring explaining what collateral is exactly?

maybe a separate ticket for proper natspecs for at least the modules makes sense. we need good docs to point users to to understand the working and usage of the modules.

Reply:
created this for the natspecs #17

Reply:
renamed the method in 376cfdc6064eaa6b5d78795ce665236b88951d1b